dunereco/HitFinderDUNE/TimeBasedDisambig.cxx

```diff
@@ -470,7 +470,8 @@ void TimeBasedDisambig::RunDisambig( const std::vector< art::Ptr<recob::Hit> > &
 		       InductionAll[uv0].RMS,
 		       InductionAll[uv0].PeakAmplitude,
 		       InductionAll[uv0].SigmaPeakAmplitude,
-		       InductionAll[uv0].SummedADC,
+		       InductionAll[uv0].SummedADC,//ROI
```

Comment:
Did this change the number or arguments?

Reply:
It did not change the number of arguments for "(std::vector<HitPos> InductionAll)" but an additional argument was needed in the hit definition so "InductionAll[uv0].SummedADC" was used twice

Reply:
@tomjunk may I merge this today?  I think I need it to build with the new LArSoft version.